### PR TITLE
Document StatDrop

### PR DESCRIPTION
**Pull request by nanaian**

### CODE REVIEW

File: include/map.h [hunk 1]
```diff
@@ -135,8 +148,8 @@ typedef struct StaticNPC {
     /* 0x028 */ s8 dropFlags;
     /* 0x029 */ s8 itemDropChance; // %
     /* 0x02A */ ItemDrop itemDrops[8];
-    /* 0x05A */ StatDrop heartDrops[8];
-    /* 0x09A */ StatDrop flowerDrops[8];
+    /* 0x05A */ StatDrop[5] heartDrops;
```
Comment (ethteck) [inline include/map.h:151]: We've been putting variable sizes after the name

Comment (nanaian) [reply include/map.h:151]: oh, whoops, typo

Comment (nanaian) [reply include/map.h:151]: I was going to change `typedef StatDrop -> StatDrops[5]` but decided against it

**Comment by ethteck** [comment]
Somehow this PR is non-matching.. lol

File: include/map.h [hunk 2]
```diff
@@ -135,8 +148,8 @@ typedef struct StaticNPC {
     /* 0x028 */ s8 dropFlags;
     /* 0x029 */ s8 itemDropChance; // %
     /* 0x02A */ ItemDrop itemDrops[8];
-    /* 0x05A */ StatDrop heartDrops[8];
-    /* 0x09A */ StatDrop flowerDrops[8];
+    /* 0x05A */ StatDrop heartDrops[5];
```
Comment (ethteck) [inline include/map.h:151]: Oh, this branch isn't matching because you changed the size of this struct

Comment (nanaian) [reply include/map.h:151]: Whoops 2

**Review by pixel-stuck — APPROVED** [review]
this approve is pending correction of those sizes of course.

**Pull request merged**